package com.test.exception;

public class TestDemoException {
	//static ArithmeticException e=new ArithmeticException();
	public static void main(String[] args) throws CustomException {
		int a = 10;
		int b = 20;
//		if(a<b) {
//			throw new CustomException("a value should be greater than b");
//		}
//		if(a<b) {
//			throw new CustomException();
//		}
		// TODO Auto-generated method stub
	//	System.out.println("State - 1");
		try {
			System.out.println(10/0);
		}catch(ArithmeticException e){
			System.out.println(e.getStackTrace(););
			
			//System.out.println(e);
			System.out.println(e.getMessage());
		}
		
		//System.out.println("State - 2");
	}

}
